CMSgov/mdct-mcr
# MDCT-94: Configure Cypress testing

# Description

The [ticket](https://qmacbis.atlassian.net/browse/MDCT-94) was just to set up Cypress testing in the repo, which it turns out was already done thanks to QMR. There was a bunch of stuff that needed to be ripped out and cleaned up, though.

## How to test


To run integration tests: 
  - in project root dir
  - `yarn && yarn test`
  - select `commonPages.spec.js` in the Cypress window that appears
  - watch the testing occur

Note: There are some minor accessibility issues that will be fixed later when we actually have designs built for MCR. You'll also see some API call failures, which are not a big deal at this point because that will also be changed later. All that's really being tested is the login and homepage render. 👍

## Dependencies

No updated dependencies.

# Get it done


1. You will review it (thanks!)
2. I will merge it (good job, me).

## Code authors checklist


## Reviewers Checklist (Two different people)


## Assignee
